nifi-stateless/nifi-stateless-core/src/main/java/org/apache/nifi/stateless/core/StatelessRemoteOutputPort.java

```diff
@@ -139,7 +140,11 @@ public boolean runRecursive(final Queue<InMemoryFlowFile> queue) {
 
             transaction.confirm();
             transaction.complete();
-        } catch (final Exception e) {
+        } catch (final NoContentException e) {
+            getLogger().info("No flowfiles to receive");
```

Comment:
Not sure if `info` logging is intended. In other places the "No flowfile" case only warranted a debug level log.

Reply:
Good spot, will change.